src/QuoteCard/QuoteCard.tsx

```diff
@@ -7,12 +7,13 @@ import { QuoteInterface } from "../interfaces";
 interface QuoteCardProps {
   quote: string;
   author: string;
+  image: string;
```

Comment:
The images add such a nice touch to the cards! Great work! Love that we can just pull them from the API.